### PR TITLE
Don't install dgoss on default

### PR DESCRIPTION
**Pull request by t2d**

### CODE REVIEW

**Comment by bbaassssiiee** [comment]
What problem does this fix?

**Comment by t2d** [comment]
I don't need this unknown binary on my servers. So I don't want it there.

**Comment by bbaassssiiee** [comment]
It's a shell script to test containers

**Comment by t2d** [comment]
Still, I don't need it and don't want it on the system. Do you want me to change the variable default so that it keeps your default behavior? Or do you not want this PR in general?

**Comment by bbaassssiiee** [comment]
Please toggle the default to keep the current behaviour, it can then be set to false for people that don't want dgoss.

File: defaults/main.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ goss_url: "https://github.com/aelsabbahy/goss/releases/download/{{ goss_version
 goss_test_directory: /root
 goss_test_directory_mode: 0700
 goss_format: tap
+goss_install_dgoss: false
```
Comment (bbaassssiiee) [inline defaults/main.yml]: true please

Comment (t2d) [reply defaults/main.yml]: done

**Comment by bbaassssiiee** [comment]
TASK [base_goss : copy dgoss to remote] ****************************************
[DEPRECATION WARNING]: evaluating goss_install_dgoss as a bare variable, this 
behaviour will go away and you might need to add |bool to the expression in the
 future. Also see CONDITIONAL_BARE_VARS configuration toggle.. This feature 
will be removed in version 2.12. Deprecation warnings can be disabled by 
setting deprecation_warnings=False in ansible.cfg.

**Comment by t2d** [comment]
I added the filter you asked for.
````
- name: "copy dgoss to remote"
  copy:
    src: "dgoss"
    dest: "{{ goss_path }}/dgoss"
    mode: 0755
  no_log: yes
  when: goss_install_dgoss|bool
  tags:
    - base_goss
```

**Comment by bbaassssiiee** [comment]
Please update this branch to resolve the conflicts.

**Comment by t2d** [comment]
> Please update this branch to resolve the conflicts.
Done

**Pull request merged**